staging/src/k8s.io/component-base/metrics/features/kube_features.go

```diff
@@ -17,26 +17,11 @@ limitations under the License.
 package features
 
 import (
-	"k8s.io/apimachinery/pkg/util/version"
 	"k8s.io/component-base/featuregate"
 )
 
-const (
-	// owner: @logicalhan
-	// kep: https://kep.k8s.io/3466
-	ComponentSLIs featuregate.Feature = "ComponentSLIs"
-)
-
 func featureGates() map[featuregate.Feature]featuregate.VersionedSpecs {
-	return map[featuregate.Feature]featuregate.VersionedSpecs{
-		ComponentSLIs: {
-			{Version: version.MustParse("1.26"), Default: false, PreRelease: featuregate.Alpha},
-			{Version: version.MustParse("1.27"), Default: true, PreRelease: featuregate.Beta},
-			// ComponentSLIs officially graduated to GA in v1.29 but the gate was not updated until v1.32.
-			// To support emulated versions, keep the gate until v1.35.
```

Comment:
The comment here suggests to keep the gate till v1.35 to support emulated versions. Can we remove the gate just yet? @Jefftree

Reply:
We're in 1.35 now right?